### DISCUSSION TITLE
S7 connection error readInt(5,2,16,False): b‘ TCP: unreachable peer‘@line 607

### DISCUSSION THREAD

**Discussion by MarioGiesenW15** [artisan-roaster-scope/artisan#894] [Q&A]
Who can help? Computer doesn’t connect to my Giesen W15 anymore 😩
![242BA0FB-4499-41EF-BBB3-58DCD2BA7DA3](https://user-images.githubusercontent.com/106835235/171923824-c0bb2084-6d5b-44ad-9b6d-6cb482fe9dfa.png)

**Comment by poundy** [comment]
Giesen probably? 

Want to help us out by telling us the story here? Has this ever worked and it's just stopped, or have you just commissioned this machine and are still trying to get it working? Have you ever had any other roasting software talking to the roaster? 

Having said that, to me this is a pretty clear error - the roaster, over TCP, is unreachable.  If you read the machine info page https://artisan-scope.org/machines/giesen/ it says that you have the network configured specifically to allow communication. Have you followed that ? Have you confirmed that your roaster has the default IP address listed, if that's how you set it up?

**Reply by MarioGiesenW15** [reply]
Hi poundy, thank you for your answer.

Yes, connection has worked before. It popped up two weeks ago. All of a sudden I was not able to connect them anymore.

I followed the instructions given, setting up the up addresses as suggested.

I‘ve done the Artisan update…. Still nothing :( 

There was no other software installed so far. I‘ve asked everybody I know using Giesen and Artisan. No answer so far… Yesterday I’ve written to Giesen…. Maybe they have an idea?!

**Reply by aqdennis** [reply]
It can't be something silly as a bad network cable? Maybe it's making bad contact or so ..
Furthermore, could there have been an OS update that caused this?

**Reply by MarioGiesenW15** [reply]
OS? I don’t use a Mac, normal Windows PC

I don’t know :( so frustrating

**Reply by aqdennis** [reply]
Exactly....and Windows has the habit of running background updates. If you have excluded the cable you may consider checking if you have a Windows restore point dated just before it went wrong and if so, restore to that point. If that solves it the problem is a Windows update...

**Reply by poundy** [reply]
Great to know that the combination has worked in the past. Have you verified the IP address on the Giesen hasn't changed ?  I have no idea if you can see that on a control panel, but for this I'd try to go back to network basics, can you PING the roaster IP address from the Windows PC, as if you can't (or get a message about network unreachable) then it's likely a configuration thing on the PC... 
Simplest next step here is to tell us what your PC IP setup is, and tell us what the IP of your roaster is. If you open a command prompt or powershell prompt or Windows terminal (depending on your version of Windows) you can type IPCONFIG and you will see on one of your network interfaces the IP address ("IPv4 Address"), subnet mask, and default gateway - relay those three things to us, plus your roaster's IP address.

**Reply by MarioGiesenW15** [reply]
All sorted…. Giesen helped via TeamViewer. Thank you all for your help

**Reply by poundy** [reply]
great news !  Any chance you can give us a quick rundown of what was wrong, to help if it comes up in the future?
